lib/prng/fastrandombytes.cpp

```diff
@@ -4,44 +4,29 @@
  * Public Domain
  */
 
-#include "nfl/prng/crypto_stream_salsa20.h"
-#include "nfl/prng/randombytes.h"
 #include <inttypes.h>
 #include <iostream>
+#include "nfl/prng/crypto_stream_salsa20.h"
+#include "nfl/prng/randombytes.h"
+
+namespace nfl {
 
 static int init = 0;
-static unsigned char key[crypto_stream_salsa20_KEYBYTES];
-static unsigned char nonce[crypto_stream_salsa20_NONCEBYTES] = {0};
+static unsigned char key[32];
```

Comment:
I just don't know whether they really needed to be in the first place. Would we gain anything by defining symbolic values?

Also in
`for (i = 0; i < 8; i++) nonce[i] = (n >> 8 * i) & 0xff;`
the value `crypto_stream_salsa20_NONCEBYTES ` was hardcoded as 8.